src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -60,7 +60,16 @@ public function registerClasses(Definition $prototype, $namespace, $resource, $e
         $interfaces = array();
         $singlyImplemented = array();
 
-        foreach ($classes as $class) {
+        foreach ($classes as $class => $errorMessage) {
+            if (null !== $errorMessage) {
+                $definition = new Definition($class);
```

Comment:
This is a nit-pick, but what happens of `$class` is actually an alias? I *imagine* it would make no difference: if that Definition is ultimately used (likely due to autowiring), it would throw the error.

Reply:
I don't expect this case to be any different than with an alias to a non-broken class. But I'll test it.